activerecord/lib/active_record/enum.rb

```diff
@@ -138,19 +152,26 @@ def enum(definitions)
         _enum_methods_module.module_eval do
           pairs = values.respond_to?(:each_pair) ? values.each_pair : values.each_with_index
           pairs.each do |value, i|
+            value_prefix = \
```

Comment:
I looked at `delegate` style of prefix implementation. If it matter I'll change it.